src/main/java/uk/gov/ons/ctp/integration/rhsvc/endpoint/WebformEndpoint.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.ons.ctp.integration.rhsvc.endpoint;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import io.micrometer.core.annotation.Timed;
+import java.util.UUID;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ons.ctp.common.error.CTPException;
+import uk.gov.ons.ctp.integration.rhsvc.representation.WebformDTO;
+import uk.gov.ons.ctp.integration.rhsvc.service.WebformService;
+
+/** The REST endpoint controller for the webform endpoint. */
+@Timed
+@RestController
+@RequestMapping(value = "/", produces = "application/json")
+public final class WebformEndpoint {
+
+  private static final Logger log = LoggerFactory.getLogger(WebformEndpoint.class);
+
+  @Autowired private WebformService webformService;
+
+  @RequestMapping(value = "/webform", method = RequestMethod.POST)
+  public void webformCapture(@Valid @RequestBody WebformDTO webformDTO) throws CTPException {
+
```

Comment:
Doesn't the endpoint need to give any response? I'm guessing not but I don't understand why not?

Reply:
By default Spring returns a 200 (which the Jira ticket says that the endpoint should do).
Here is the response status from a test curl command:
`
> curl -i --data @/tmp/webformRequestDTO.json -H "Content-Type: application/json" "http://localhost:8071/webform"HTTP/1.1 200 OK
Expires: 0
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
X-XSS-Protection: 1; mode=block
Pragma: no-cache
X-Frame-Options: DENY
Date: Tue, 08 Sep 2020 08:26:20 GMT
Connection: keep-alive
X-Content-Type-Options: nosniff
Content-Length: 0
`

Reply:
Thanks for explaining Peter - that makes sense